bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -624,29 +625,35 @@ export default {
       this.eventSource = null
     },
     playAudioUrl (audioUrl) {
-      let context
       try {
-        window.AudioContext = window.AudioContext || window.webkitAudioContext
-        if (typeof (window.AudioContext) !== 'undefined') {
-          context = new AudioContext()
+        if (!this.audioContext) {
+          window.AudioContext = window.AudioContext || window.webkitAudioContext
+          if (typeof (window.AudioContext) !== 'undefined') {
+            this.audioContext = new AudioContext()
+            unlockAudioContext(this.audioContext)
+          }
         }
         console.log('Playing audio URL: ' + audioUrl)
         this.$oh.api.getPlain(audioUrl, '', '*/*', 'arraybuffer').then((data) => {
-          context.decodeAudioData(data, function (buffer) {
-            let source = context.createBufferSource()
+          this.audioContext.decodeAudioData(data).then((buffer) => {
```

Comment:
When using the Promise based version of `decodeAudioDate`, we should be aware that the Promise version was added to Safari just in 2021 (see https://developer.mozilla.org/en-US/docs/Web/API/BaseAudioContext/decodeAudioData#browser_compatibility), which will cause issues with older iOS devices. WDYT?

Reply:
ahh, was not aware of that, i'll change it shortly.

Reply:
Thx 👍

Reply:
Very good catch, fixed